test/function.localstack.test.ts

```diff
@@ -239,32 +239,54 @@ localstackTestSuite("functionStack", (testResource, _stack, app) => {
     }
   );
 
+  // Function serialization breaks when assigning an integration/construct to a variable in the closure.
+  // TODO: what should happen here?
+  testResource.skip(
+    "Call Lambda AWS SDK put event to bus with in closure reference",
+    (parent) => {
+      const bus = new EventBus<EventBusRuleInput>(parent, "bus");
+      const func = new Function(parent, "function", async () => {
+        const busbus = bus;
+        busbus({
+          "detail-type": "anyDetail",
+          source: "anySource",
+          detail: {},
+        });
+      });
+
+      return { outputs: { function: func.resource.functionName } };
```

Comment:
nit: forgetting prettier again?

Reply:
prettier runs on save and on commit. Seems like it is fine with the format. 

![image](https://user-images.githubusercontent.com/289213/167028894-bba010bb-a06d-4a83-8ac2-b4d96e242c15.png)

SAVE

![image](https://user-images.githubusercontent.com/289213/167028906-8bed7eb3-be37-4b1a-8781-0f1f2edb8861.png)


![image](https://user-images.githubusercontent.com/289213/167028927-17483a7e-cc18-484e-8d0a-08d344e7c6b8.png)


SAVE


![image](https://user-images.githubusercontent.com/289213/167028954-b4995560-8f52-4be6-8d15-9cb35b84f0c0.png)


![image](https://user-images.githubusercontent.com/289213/167029000-1d8df662-a7bf-4db6-8daa-a2ee6568c620.png)

SAVE

![image](https://user-images.githubusercontent.com/289213/167029013-8fb48fe2-76b8-4937-a6f9-1de77c1d4078.png)

Reply:
Is there something specific about the single form you don't like? Should the prettier rules be updated to reflect that?

Reply:
No worries, just looked like it wasn't formatted because everything was on one line.